engine/launcher/src/main/scala/org/enso/launcher/installation/DistributionInstaller.scala

```diff
@@ -355,6 +413,22 @@ class DistributionInstaller(
 
 object DistributionInstaller {
 
+  /**
+    * Creates a [[DistributionInstaller]] using the default managers.
+    */
+  def makeDefault(
```

Comment:
You can call this just `default`. You're using it qualified on the object anyway, so it's pretty clear.